tox.ini

```diff
@@ -15,42 +15,46 @@ deps =
     pyyaml
     pytest
     pytest-mock
-    charms.unit_test
     ipdb
+    git+https://github.com/juju-solutions/charms.unit_test/#egg=charms.unit_test
+# tox3 only passes through the upper-case versions by default, but some
+# programs, such as wget or pip, only honor the lower-case versions
+passenv =
+    http_proxy
+    https_proxy
+    no_proxy
```

Comment:
tox4 didn't like these on the same line, but newlines are [compatible with both tox3 and 4](https://github.com/tox-dev/tox/issues/2615).